COMMUNITY_MEMBERSHIP.md

```diff
@@ -95,6 +95,13 @@ Approver status can be scoped to a part of the codebase. For example, critical c
 - Sponsored by 2 maintainers. Consider the following requirements for sponsors:
   - No objections from other maintainers
   - Done through a PR that updates the corresponding `CODEOWNERS` file
+- [Open an issue](https://github.com/keptn/community/issues/new?template=membership.md&title=REQUEST%3A%20Approver%20membership%20for%20%3Cyour-GH-handle%3E)
```

Comment:
Can we try to set the title using: `title=REQUEST%3Membership%20approver%20for%3Cyour-GH-handle%3E`

+ the same for Line 54, but for `member` instead of `approver`